README.md

```diff
@@ -126,15 +126,17 @@ evm:
     keyring-pass-env-name: ETH_PASSWORD
     signing-key: ${ETH_SIGNING_KEY}
     keyring-dir: ~/.pigeon/keys/evm/eth-main
-    gas-adjustment: 1.2
+    gas-adjustment: 2 # will replace with hardcoded value (2) if txType is 2
```

Comment:
If we are replacing the value, it should be a hard error to configure an incorrect value. In general here, if we can get the program to push people towards _exactly_ what we expect in the file we will have an easier time remotely debugging.

Reply:
Replacing the value does not mean that other values are not correct. If we use too great value here, we need to prepare much funds in validator wallets. If we use too small value here, it can make tx stuck.